tests/opentracer/test_tracer_asyncio.py

```diff
@@ -7,7 +7,9 @@
 
 
 def get_dummy_asyncio_tracer():
-    return get_dummy_ot_tracer('asyncio_svc', {}, AsyncioScopeManager())
+    from ddtrace.contrib.asyncio import context_provider
+    return get_dummy_ot_tracer('asyncio_svc', {}, AsyncioScopeManager(),
+                               context_provider)
```

Comment:
can you add a test also for these context providers? it's enough one teste where you:
* create a DD span
* you yield another coroutine (so a context switch will happen)
* you create a OT span
* you assert the parenting

You can copy a simple test from `asyncio` test suite, and adapt it to the checklist above.

Reply:
True, I'll add tests that cross thread boundaries with OT/DD tracers.